src/browser/auth/Logout.react.js

```diff
@@ -4,12 +4,13 @@ import React, {PropTypes} from 'react';
 export default class Logout extends Component {
 
   static propTypes = {
+    actions: PropTypes.object.isRequired,
     msg: PropTypes.object.isRequired
   }
 
   logout() {
-    // Always reload app on logout for security reasons.
-    location.href = '/';
```

Comment:
@ondrejbartas It was better ot handle this in the component as it is browser only feature.